Tutorial_1a_Convolving_Images.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    beam = radio_beam.Beam(major=33*u.arcsec, minor=33*u.arcsec, pa=0*u.deg)
</pre><p>also note that, like the Gaussian2DKernel class, only specifying the major axis works too for a circular beam</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>